struct.go

```diff
@@ -198,24 +200,34 @@ func (s *Struct) InsertIntoForTag(table string, tag string, value interface{}) *
 		return ib
 	}
 
-	v := dereferencedValue(value)
+	var vs []reflect.Value
 
-	if v.Type() != s.structType {
+	for _, item := range value {
+		v := dereferencedValue(item)
+		if v.Type() == s.structType {
+			vs = append(vs, v)
+		}
+	}
+	if len(vs) == 0 {
 		return ib
 	}
 
 	cols := make([]string, 0, len(fields))
-	values := make([]interface{}, 0, len(fields))
+	values := make([][]interface{}, len(vs), len(vs))
 
 	for _, f := range fields {
-		name := s.fieldAlias[f]
-		data := v.FieldByName(name).Interface()
 		cols = append(cols, f)
-		values = append(values, data)
+		name := s.fieldAlias[f]
+		for i, v := range vs {
+			data := v.FieldByName(name).Interface()
```

Comment:
The `data` should be set outside the for loop as it doesn't change in the loop at all.

Reply:
I find my previous comment is not correct. You can ignore it without taking any action.